Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptEditView.jsp

```diff
@@ -4,248 +4,248 @@
-						for (var i = 0; i < len; i++) {
-							html += '<tr><td align="justify"><font size="2">'
-									+ '<a onclick="synonymRemove(\''
-									+ response[i].id + '\')">Remove</a>'
-									+ '</font></td>';
-							html += '<td align="justify"><font size="2">'
-									+ response[i].word + '</font></td>';
-							html += '<td align="justify"><font size="2">'
-									+ response[i].description
-									+ '</font></td></tr>';
-						}
-						html += '</tbody></table>';
-						$("#addedSynonyms").html(html);
-					}
-				});
-	};
+    var synonymAdd = function(synonymid) {
+        $("#dialog").dialog("close");
+        $("#synonymsDialogTable").hide();
+        $
+                .ajax({
+                    type : "GET",
+                    url : "${pageContext.servletContext.contextPath}/getConceptAddSynonyms",
+                    data : {
+                        synonymid : synonymid
+                    },
+                    success : function(response) {
+                        var synonym = JSON.parse(response);
+                        var eachSynonym = synonym.synonyms[0];
 
-	var synonymRemove = function(synonymid) {
-		$
+                        //Adding synonymId to the existing list
+                        var list = document.getElementById("synonymsids").value;
+                        list += eachSynonym.Id + ',';
+                        $("#synonymsids").val(list);
 
-				.ajax({
-					type : "GET",
-					url : "${pageContext.servletContext.contextPath}/conceptEditRemoveSynonym",
-					data : {
-						synonymid : synonymid
-					},
-					success : function(response) {
-						var border = response.length > 0 ? 1 : 0;
-						var html = '<table border="'+ border +'" width="400" id="addedSynonymsTable"><thead></thead><tbody>';
-						var len = response.length;
-						for (var i = 0; i < len; i++) {
-							html += '<tr><td align="justify"><font size="2">'
-									+ '<a onclick="synonymRemove(\''
-									+ response[i].id + '\')">Remove</a>'
-									+ '</font></td>';
-							html += '<td align="justify"><font size="2">'
-									+ response[i].word + '</font></td>';
-							html += '<td align="justify"><font size="2">'
-									+ response[i].description
-									+ '</font></td></tr>';
-						}
-						html += '</tbody></table>';
-						$("#addedSynonyms").html(html);
+                        var htmlValue = '<tr id='+eachSynonym.Id+'><td align="justify"><font size="2">'
+                                + '<a onclick="synonymTemporaryRemove(\''
+                                + eachSynonym.Id
+                                + '\')">Remove</a>'
+                                + '</font></td>';
+                        htmlValue += '<td align="justify"><font size="2">'
+                                + eachSynonym.Word + '</font></td>';
+                        htmlValue += '<td align="justify"><font size="2">'
+                                + eachSynonym.Description + '</font></td></tr>';

```

Comment:
looking at this again, I think I confused it with bootstrapTable append. You already do what I had in mind. We might want to think to switch to Bootstrap for the layout, and then we can make this much prettier. but for now, let's leave it with this.